src/vs/workbench/contrib/webview/common/webview.ts

```diff
@@ -79,3 +79,5 @@ export interface Webview extends IDisposable {
 }
 
 export const webviewDeveloperCategory = nls.localize('developer', "Developer");
+
+export const webviewEditorId = 'WebviewEditor';
```

Comment:
Having to move the editor id out of the editor is not great. Not sure if there's a better solution for this